strings/tcg/set_one.json

```diff
@@ -61,74 +61,74 @@
 			"id": "assistant",
 			"name": "Staff Assistant",
 			"desc": "The lowest ladder on the Nanotrasen Employment Ladder, Staff Assistants are employed to help out with tasks deemed 'too menial for robots'.",
-			"rules": "{$Graytide}, for every card with '{$Graytide}', this card has +1/+1 on the field.",
+			"rules": "{$Graytide}. The buff this card gains from Graytide is doubled.",
 			"icon_state": "assistant",
 			"power": "1",
 			"resolve": "1",
 			"faction": "Service",
 			"summoncost": "1",
 			"cardtype": "Creature",
-			"cardsubtype": "Human Employee",
+			"cardsubtype": "Maintenance Service",
 			"rarity": "common",
 			"summon_icon_state": "Assistant"
 		},
 		{
 			"id": "atmos_tech",
 			"name": "Atmospheric Technician",
 			"desc": "The Atmospheric Technicians are tasked with keeping the station's air clean, breathable, and, most importantly, devoid of plasma.",
-			"rules": "{$On Summon}: Search your deck for an Atmospherics Battlefield card, and add it to your hand. Shuffle your deck afterward.",
+			"rules": "{$On Summon}: Search your deck for a Battlefield card, and add it to your hand. Shuffle your deck afterward.",
 			"icon_state": "atmos_tech",
-			"power": "2",
-			"resolve": "3",
+			"power": "3",
+			"resolve": "2",
 			"faction": "Engineering",
-			"summoncost": "4",
+			"summoncost": "2",
 			"cardtype": "Creature",
-			"cardsubtype": "Human Engineer",
+			"cardsubtype": "Atmosian Engineering",
```

Comment:
This is more of an EOB thing, but considering that within the common core this would be published material from Centcom, Nanotrasen would probably not use slang terms for departments on their printed ~~propaganda~~ materials. paging @EOBGames